src/main/java/org/cactoos/scalar/ItemAt.java

```diff
@@ -27,74 +27,71 @@
 import java.util.Iterator;
 import org.cactoos.Func;
 import org.cactoos.Scalar;
+import org.cactoos.func.FuncOf;
 import org.cactoos.text.FormattedText;
 
 /**
- * Element from position in {@link Iterator}
- * or fallback value if iterator hasn't this position.
+ * Element from position in {@link Iterable}
+ * or fallback value if iterable hasn't this position.
  *
  * <p>There is no thread-safety guarantee.
  *
  * @param <T> Scalar type
  * @since 0.7
- * @todo #930:30min To be consistent with other objects working on iterables
- *  such as HeadOf, TailOf, etc, the constructor of ItemAt should always take
- *  the iterable/iterator as a last parameter.
  */
 public final class ItemAt<T> implements Scalar<T> {
 
     /**
-     * {@link Sticky} that holds the value of the iterator
+     * {@link Sticky} that holds the value of the iterable
      *  at the position specified in the constructor.
      */
     private final Scalar<T> saved;
 
     /**
      * Ctor.
      *
-     * @param source Iterable
+     * @param iterable Iterable
      */
-    public ItemAt(final Iterable<T> source) {
+    public ItemAt(final Iterable<T> iterable) {
         this(
-            source,
             itr -> {
                 throw new IOException("The iterable is empty");
-            }
+            },
+            iterable
         );
     }
 
     /**
      * Ctor.
      *
      * @param fallback Fallback value
-     * @param source Iterable
+     * @param iterable Iterable
      */
-    public ItemAt(final T fallback, final Iterable<T> source) {
-        this(source, itr -> fallback);
+    public ItemAt(final Scalar<T> fallback, final Iterable<T> iterable) {
```

Comment:
@victornoel this ctor doesn't make sense because it's missing the index parameter. Please remove it